sw/device/rom_exts/rom_ext_manifest_parser.h

```diff
@@ -41,6 +41,32 @@ typedef struct rom_ext_manifest {
   rom_ext_manifest_slot_t slot;
 } rom_ext_manifest_t;
 
+/**
+ * ROM Extension Memory Extents.
+ *
+ * This information is needed for calculating the signature of the entire
+ * ROM_EXT.
+ */
+typedef struct rom_ext_extents {
+  /**
+   * Image Start Address
+   */
+  uintptr_t image_start;
+  /**
+   * Signed Area Start Address
+   */
+  uintptr_t signed_area_start;
```

Comment:
Could you add some more detailed information about these members (or, include a link to documentation that explains them in the struct comment)?

Reply:
I've done some of this, and added a reference at the top to the place where the format is documented in its entirety.